src/ess/imaging/data.py

```diff
@@ -4,19 +4,21 @@
 
 import pooch
 
-_version = '0'
+_version = '1'
 
 
 def _make_pooch():
     return pooch.create(
         path=pooch.os_cache('essimaging'),
         env='BEAMLIME_DATA_DIR',
         retry_if_failed=3,
-        base_url='https://public.esss.dk/groups/scipp/ess/imaging/',
+        base_url=f'https://public.esss.dk/groups/scipp/ess/imaging/{_version}/',
         version=_version,
         registry={
+            'small_mcstas_ob_images.h5': 'md5:2f181bbacb164c28bfaf7cce09701d92',
+            'small_mcstas_sample_images.h5': 'md5:3c42570951cabec7caedc76d90d03fa3',
             'small_ymir_images.hdf': 'md5:cf83695d5da29e686c10a31b402b8bdb',
-            'README.md': 'md5:0f375972d4008de6060b065ac13ba17f',
+            'README.md': 'md5:9e1beeb325f127d691a8d7882db3255d',
```

Comment:
looks like there is no README anymore? https://public.esss.dk/groups/scipp/ess/imaging/1/

The tests are failing.

Reply:
@MridulS 
<img width="415" alt="image" src="https://github.com/user-attachments/assets/70268601-6f3c-4d3c-9c6d-5dc5662cf035">

It is there. It just doesn't show up in the web-thingy. 

I'll see why it's failing though. It worked last time I made this change.

Reply:
Which tests are failing btw?

Reply:
https://github.com/scipp/essimaging/actions/runs/11770808535/job/32783786918

Reply:
Ahhh, the failure should go away after a release. The failure is at latest release which doesn't take into account the new url (with _version) and trying to find README files at the old location. Ignore me :)